app/Resources/views/legislatives/contact.html.twig

```diff
@@ -0,0 +1,89 @@
+{% extends 'base.html.twig' %}
+
+{% block page_title 'Contact | Législatives' %}
+{% block canonical_url url('app_legislative_candidates_platform_contact') %}
+
+{% block content %}
+    <article class="l__wrapper--slim b__nudge--bottom-huge l__col l__col--center">
+        <h1>Vous avez une question ?</h1>
+        <h2>Contactez-nous</h2>
+        <div class="register__form">
+            <form action="{{ path('app_legislative_candidates_platform_contact') }}" method="post" novalidate="novalidate" id="legislatives_campaign_contact_form">
```

Comment:
`novalidate` because I wanted to test in the web browser without having it preventing me from submitting the form with blank values. I forgot to remove it.

I don't like the `form_start()` function because it forces to also use the `form_end()` function as well to keep the markup valid in the IDE.